<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget {
	background: #262626;
}

QTabBar {
	background: #262626;
}

QTabBar::tab {
	background: none;
	border: none;
	margin: 0px;
	color: lightgray;
	width: 170px;
	height: 24px;
}

QTabBar::tab:hover {
	background: #333333;
}

QTabBar::tab:selected {
	background: #214478;
	color: white;
}

QTabWidget::pane {
	background: #1a1a1a;
	border: none;
}

QLabel {
	color: lightgray;
	min-width: 120px;
}

QScrollArea {
	border: none;
}

QScrollArea &gt; QWidget &gt; QWidget {
	background: #1a1a1a;
}</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::West</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="tabGeneral">
   <attribute name="title">
    <string>General</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="tabOutput">
   <attribute name="title">
    <string>Output</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>412</width>
         <height>282</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="maximum">
           <double>16000.000000000000000</double>
          </property>
          <property name="value">
           <double>240.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="maximum">
           <double>16000.000000000000000</double>
          </property>
          <property name="value">
           <double>320.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>px</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>in</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pc</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox_2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>px/cm (DPC)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>px/in (DPI)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="value">
           <double>72.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" rowspan="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabSave">
   <attribute name="title">
    <string>Save</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>412</width>
         <height>282</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>untitled</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabAntialiasing">
   <attribute name="title">
    <string>Anti-Aliasing</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="tabOptions">
   <attribute name="title">
    <string>Options</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
